src/common/utils/email/mail.go

```diff
@@ -117,24 +118,77 @@ func sendMailWithTLS(m Mail, auth smtp.Auth, content []byte) error {
 	return client.Quit()
 }
 
-/*
-func loadConfig() {
-	config, err := beego.AppConfig.GetSection("mail")
+// Ping tests the connection and authentication with email server
+// If tls is true, a secure connection is established, or the
+// connection is insecure, and if starttls is true, Ping trys to
+// upgrate the insecure connection to a secure one if email server
+// supports it.
+// Ping doesn't verify the server's certificate and hostname when
+// needed if the parameter insecure is ture
+func Ping(addr, identity, username, password string,
+	timeout int, tls, starttls, insecure bool) (err error) {
```

Comment:
Maybe trivial.
I'm not sure if it's my personal taste....
but I would prefer the form:
```
 func xxx() error{
    err := xxxxx
    return err
}
```